src/main/java/bio/terra/service/configuration/ConfigFault.java

```diff
@@ -0,0 +1,79 @@
+package bio.terra.service.configuration;
+
+import bio.terra.app.controller.exception.ValidationException;
+import bio.terra.model.ConfigFaultModel;
+import bio.terra.model.ConfigModel;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+// Sub-classes must implement:
+// get()
+// setFaultConfig()
+public abstract class ConfigFault extends ConfigBase {
+    private final Logger logger = LoggerFactory.getLogger(ConfigFault.class);
+
+    private final ConfigFaultModel.FaultTypeEnum faultType;
+    private final boolean originalEnabled;
```

Comment:
Is this property intended to be used anywhere except for reset()? If not, I think it's slightly less confusing to have the "original state" not include the enabled status. So reset() would either not change enabled, or always change it in a consistent way (e.g. always disabling it). However, I understand this doesn't quite match the reset() behavior of other non-fault configs, so I can see doing it this way too. Just adding my two cents.

Reply:
When the system starts there can be faults that are enabled by default. I want reset() to return the system to that base state; that is, to have overall consistent behavior.

It is rare to have a fault that is enabled in the initial state, but if you want to insert faults early in the life of the system, it can be handy.